### PR TITLE
CNF-13591: bundle: Add infrastructure annotations for midstream builds

### PR DESCRIPTION
**Pull request by leo8a**
@browsell @donpenney (when available) PTL to the bundle description, as well as the proposed annotations for this operator.

/cc @rauhersu @fontivan

image

### CODE REVIEW

File: bundle/manifests/oran-o2ims.clusterserviceversion.yaml [hunk 1]
```diff
@@ -135,8 +135,27 @@ metadata:
         }
       ]
     capabilities: Basic Install
+    containerImage: quay.io/openshift-kni/oran-o2ims
```
Comment (donpenney) [inline bundle/manifests/oran-o2ims.clusterserviceversion.yaml:138]: With https://github.com/openshift/release/pull/54191 and https://github.com/openshift-kni/oran-o2ims/pull/187, the remaining image is oran-o2ims-operator

```suggestion
    containerImage: quay.io/openshift-kni/oran-o2ims-operator
```

File: config/manifests/bases/oran-o2ims.clusterserviceversion.yaml [hunk 2]
```diff
@@ -4,6 +4,25 @@ metadata:
   annotations:
     alm-examples: '[]'
     capabilities: Basic Install
+    containerImage: quay.io/openshift-kni/oran-o2ims
```
Comment (donpenney) [inline config/manifests/bases/oran-o2ims.clusterserviceversion.yaml:7]:
```suggestion
    containerImage: quay.io/openshift-kni/oran-o2ims-operator
```

File: vendor/github.com/openshift-kni/oran-o2ims/api/hardwaremanagement/v1alpha1/node.go [hunk 3]
```diff
@@ -26,15 +26,15 @@ type NodeSpec struct {
 }
 
 type BMC struct {
-	// Address contains the URL for accessing the BMC over the network.
+	// The Address contains the URL for accessing the BMC over the network.
```
Comment (donpenney) [inline vendor/github.com/openshift-kni/oran-o2ims/api/hardwaremanagement/v1alpha1/node.go:29]: The changes in the vendor folder are from a previous update that updated the API files but did not subsequently do a "go mod vendor". Given that this is unrelated to your change, I'd prefer to see that fixed that in a separate PR. We can also setup the CI test to check the git tree and protect against this happening again

Comment (leo8a) [reply vendor/github.com/openshift-kni/oran-o2ims/api/hardwaremanagement/v1alpha1/node.go:29]: agree

Comment (donpenney) [reply vendor/github.com/openshift-kni/oran-o2ims/api/hardwaremanagement/v1alpha1/node.go:29]: https://github.com/openshift-kni/oran-o2ims/pull/188

**Review by donpenney — APPROVED** [review]

**Pull request merged**